paddle/fluid/framework/ir/fc_gru_fuse_pass.cc

```diff
@@ -48,7 +48,8 @@ static int BuildFusion(Graph* graph, const std::string& name_scope,
 
   // Create New OpDesc
   auto gru_creater = [&](Node* gru, Node* x, Node* weight_x, Node* weight_h,
```

Comment:
I found the name "gru_creater" a bit confusing. Maybe you could use "gru_creator" ?

Reply:
Oh yes, it could be a spelling mistake.